docs/abandoned-projects.rst

```diff
@@ -0,0 +1,105 @@
+Policy for Abandoned Projects
+=============================
+
+This policy describes the process by which a RTD project name may be changed.
+
+
+Rationale
+---------
+
+Conflict between the current use of the name and a different suggested use of
+the same name occasionally arise.  This document aims to provide general
+guidelines for solving the most typical cases of such conflicts.
+
+
+Specification
+-------------
+
+The main idea behind this policy is that RTD serves the community.  Every
+user is invited to upload content under the Terms of Use, understanding that it
+is at the sole risk of the user.
+
+While RTD is not a backup service, the maintainers do their best to keep that
+content accessible indefinitely in its published form.  However, in certain
+edge cases the greater community's needs might outweigh the individual's
+expectation of ownership of a project name.
+
+The use cases covered by this policy are:
+
+* Abandoned projects:
+
+    * renaming a project so that the original project name can be used by a
+      different project.
+
+* Active projects:
+
+    * resolving disputes over a name.
+
+
+Implementation
+--------------
+
+Reachability
+~~~~~~~~~~~~
+
+The user of RTD is solely responsible for being reachable by the maintainers
+for matters concerning projects that the user owns.  In every case where
+contacting the user is necessary, the maintainers will try to do so at least
+three times, using the following means of contact:
+
+* e-mail address on file in the user's profile;
+* e-mail addresses found in the given project's documentation; and
+* e-mail address on the project's home page.
+
+The maintainers will stop trying to reach the user after six weeks and the user
+will be considered *unreachable*.
+
+
+Abandoned projects
+~~~~~~~~~~~~~~~~~~
+
+A project is considered *abandoned* when ALL of the following are met:
+
+* owner is unreachable (see Reachability above);
+* no releases within the past twelve months; and
+* no activity from the owner on the project's home page (or no home page
+  found).
+
+All other projects are considered *active*.
```

Comment:
???

Reply:
Is the question "Aren't there a lot of other ways a project could qualify as inactive?"  If so, yes, but as far as a policy goes, it would seem that being strictly binary is the easiest thing for people to understand.